bin/check-memory-percent.rb

```diff
@@ -3,4 +3,4 @@
 bin_dir = File.expand_path(File.dirname(__FILE__))
 shell_script_path = File.join(bin_dir, File.basename($PROGRAM_NAME, '.rb') + '.sh')
 
-exec shell_script_path, *ARGV
+exec shell_script_path, *ARGV, :unsetenv_others => true
```

Comment:
Before:
```
asia ~ # opt/sensu/embedded/bin/check-memory-percent.rb 
awk: Kommandozeile:1: Fatal: Division durch Null wurde versucht
/opt/sensu/embedded/lib/ruby/gems/2.3.0/gems/sensu-plugins-memory-checks-3.0.1/bin/check-memory-percent.sh: Zeile 63: 100-: Syntax Fehler: Operator erwartet. (Fehlerverursachendes Zeichen ist \"-\").
MEM UNKNOWN -
```
After:
```
asia ~ # /opt/sensu/embedded/bin/check-memory-percent.rb 
MEM OK - system memory usage: 7%
```

Root cause is a call to free | awk that does not fit if the result is not en_US:
```
asia ~ # free
              gesamt       benutzt     frei      gemns.  Puffer/Cache verfügbar
Speicher:     4052940      165480     1821728       13876     2065732     3809360
Swap:             0           0           0
```